### PR TITLE
Batch - Update test with expected BYOS values

### PR DESCRIPTION
**Pull request by jafreck**

### CODE REVIEW

File: src/SDKs/Batch/Management/Management.Batch.Tests/ScenarioTests/AccountTests.ScenarioTests.cs [hunk 1]
```diff
@@ -171,8 +171,10 @@ public async Task BatchAccountCanCreateWithBYOSEnabled()
                     // Get the account and verify some properties
                     BatchAccount batchAccount = await this.BatchManagementClient.BatchAccount.GetAsync(resourceGroupName, batchAccountName);
                     Assert.Equal(batchAccountName, batchAccount.Name);
-                    Assert.True(batchAccount.DedicatedCoreQuota > 0);
-                    Assert.Equal(0, batchAccount.LowPriorityCoreQuota);
+                    Assert.True(batchAccount.DedicatedCoreQuota == null);
```
Comment (matthchr) [inline src/SDKs/Batch/Management/Management.Batch.Tests/ScenarioTests/AccountTests.ScenarioTests.cs]: Use `Assert.Null`

Comment (matthchr) [reply src/SDKs/Batch/Management/Management.Batch.Tests/ScenarioTests/AccountTests.ScenarioTests.cs]: Same for below

**Review by matthchr — APPROVED** [review]

File: src/SDKs/Batch/Management/Management.Batch.Tests/ScenarioTests/AccountTests.ScenarioTests.cs [hunk 2]
```diff
@@ -171,8 +171,10 @@ public async Task BatchAccountCanCreateWithBYOSEnabled()
                     // Get the account and verify some properties
                     BatchAccount batchAccount = await this.BatchManagementClient.BatchAccount.GetAsync(resourceGroupName, batchAccountName);
                     Assert.Equal(batchAccountName, batchAccount.Name);
-                    Assert.True(batchAccount.DedicatedCoreQuota > 0);
-                    Assert.Equal(0, batchAccount.LowPriorityCoreQuota);
+                    Assert.Null(batchAccount.DedicatedCoreQuota);
+                    Assert.Null(batchAccount.DedicatedCoreQuotaPerVMFamily );
```
Comment (matthchr) [inline src/SDKs/Batch/Management/Management.Batch.Tests/ScenarioTests/AccountTests.ScenarioTests.cs]: super minor: You have an extra trailing space here

**Review by matthchr — APPROVED** [review]

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**